app/views/expenses/new.html.erb

```diff
@@ -0,0 +1,23 @@
+<nav class="navbar navbar-expand-md navbar-dark bg-primary fixed-top">
+  <%= link_to image_tag('back.png', alt: "back-arrow"), group_expenses_path,  class: "px-2"%>
+  <h3 class="text-white p-3">Add Expense</h3>
+  <%= link_to "Logout", destroy_user_session_path, data: { turbo_method: :delete }, class: "text-white fw-bold text-decoration-none p-3" %>
+</nav>
+
+<p style="color: green"><%= notice %></p>
+<div class="container mt-5">
+<%= form_with model: @expense, url: group_expenses_url, method: :post do |form| %>
+
+  <div>
+    <%= form.text_field :name, placeholder: "Name", class:"form-control py-3 mt-3" %>
+  </div>
+
+  <div>
+    <%= form.text_field :amount, placeholder: "Amount", class:"form-control py-3 mt-3" %>
+  </div>
+
+  <div>
```

Comment:
[REQUIRED] Kindly include an input field to select which category a new expense belongs to

Reply:
Hello @fobadara , thanks for the review. The new expense is added under a category, so when the form is submitted the category id on the URL is saved in the database. I did not see a need to have a category field yet the transaction is already under a category. Thanks